web-react-ts/src/pages/arrivals/BacentaArrivals.tsx

```diff
@@ -73,7 +73,7 @@ const BacentaArrivals = () => {
   useEffect(() => handleClose(), [])
 
   const END_TIME_IN_MS = new Date(
-    getTodayTime(bacenta?.stream.arrivalEndTime)
+    getTodayTime(bacenta?.stream?.arrivalEndTime)
```

Comment:
Is there a reason why the arrivalEndTime would not be there for a stream? And if so which stream is this? I would like to know